distribution/build.gradle

```diff
@@ -285,6 +285,14 @@ configure(subprojects.findAll { ['deb', 'rpm'].contains(it.name) }) {
       we see it. We'll add it back on to the file name though. */
     version project.version.replace('-SNAPSHOT', '')
 
+    // signing setup
+    signingKeyId = project.hasProperty('signing.keyId') ? project.getProperty('signing.keyId') : System.getProperty('signing.keyId', 'D88E42B4')
```

Comment:
just using gradle properties now (I think we should port `build.snapshot` over as a gradle property as well, but not in this PR). Can you explain what is bad about setting our signing key as the default?